tests/browseruse/test_dashboard.py

```
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+"""
+Dashboard functionality tests for Maple application.
+These tests verify core dashboard features after successful authentication.
+"""
+
+import asyncio
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)